[OV JS][DOCS] Edits in JS API docs (#26875)
openvinotoolkit/openvino: src/bindings/js/node/README.md

# OpenVINO™ Node.js Bindings

Execute deep learning models in Node.js applications with ease using OpenVINO™.

## Introduction

OpenVINO™ is an open-source toolkit designed for high-performance deep learning inference.
Node.js API provides bindings to subset APIs from OpenVINO Runtime.

> These Node.js bindings allow JavaScript developers to use the power of OpenVINO™ in their applications.

Reviewer: ```suggestion
The Node.js bindings enable JavaScript developers to use the capabilities of OpenVINO in their applications.
```

const { addon: ov } = require('openvino-node');
```

> Full `addon` API you can find at documentation [page](https://docs.openvino.ai/2024/api/nodejs_api/addon.html).

Reviewer: ```suggestion
Refer to the complete description of the `addon` API in the [documentation](https://docs.openvino.ai/2024/api/nodejs_api/addon.html).
```

> See more samples in [samples section](https://github.com/openvinotoolkit/openvino/blob/master/samples/js/node/README.md).

Reviewer: ```suggestion
See the [samples](https://github.com/openvinotoolkit/openvino/blob/master/samples/js/node/README.md) for more details on how to use it.
```

## Usage in Electron applications

To use the package in development of Electron applications on Windows, make sure that
**Desktop development with C++** component from
[Build Tools for Visual Studio](https://aka.ms/vs/17/release/vs_BuildTools.exe) is installed.

## Supported Platforms

- Windows x86

> - Linux x86/arm
> - MacOS x86/arm

Reviewer: ```suggestion
- Linux x86/ARM
- MacOS x86/ARM
```

## Live Sample

> You can run this sample in the browser, no installation is required.

Reviewer: ```suggestion
You can run the following sample in the browser, no installation is required.
```

[Codesandbox](https://codesandbox.io/) is a free online service with limited resources. For optimal performance and more control,  it is recommended to run the sample locally.

- [hello-classification-sample](https://codesandbox.io/p/devbox/openvino-node-hello-classification-sample-djl893)

## Build From Sources

For more details, refer to the [OpenVINO™ JavaScript API Developer Documentation](https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/js/docs/README.md#openvino-node-package-developer-documentation)

## Contributing

> We welcome contributions! Please read our [Contributing Guide](https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/js/node/CONTRIBUTING.md) to learn how you can get involved.

Reviewer: ```suggestion
Contributions are always welcome! Read the [Contribution Guide](https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/js/node/CONTRIBUTING.md) to learn how you can get involved.
```
